src/bin/uefi.rs

```diff
@@ -1,7 +1,8 @@
 #![no_std]
 #![no_main]
 #![feature(abi_efiapi)]
-#![feature(maybe_uninit_extra)]
+#![feature(cfg_version)]
+#![cfg_attr(not(version("1.60")), feature(maybe_uninit_extra))]
```

Comment:
I think we can just remove the `maybe_uninit_extra` feature unconditionally. Would you mind opening a separate PR for that?